backend/scripts/test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Runs blnd-tender tests.
+# All passed parameters go to --test-arguments parameter of the test executable.
+
+# Dependencies:
+# * Postgres server
+# * pg_tmp (http://eradman.com/ephemeralpg/)
+
+# Prerequisites:
+# * Specify your database user name via 'PGUSER' variable; default user name is your OS username.
+#   If you haven't created a database user yet, run
+#   > sudo -u postgres createuser --interactive --pwprompt
+# * If your database user's password is not empty, specify it in .pgpass file
+#   (see https://www.postgresql.org/docs/9.3/libpq-pgpass.html).
+
+set -e -o pipefail
+
+if ! which initdb > /dev/null; then
+    echo -e "Command 'initdb' not found.\nConsider adding '/usr/lib/postgresql/VERSION/bin/' to your PATH."
+    exit 2
+fi
+
+# A temporal server does not live long (60 sec by default), building tests first.
+stack test --no-run-tests
```

Comment:
I think it's better to use `make` here because it also passes `--fast` and may pass other flags we find essential for dev build.

Reply:
Maybe in this case we can run tests from make and say to build before tests?